x/derivatives/types/positions.go

```diff
@@ -302,28 +305,34 @@ func (m PerpetualFuturesPosition) ProfitAndLossInQuote(baseMetricsRate, quoteMet
 
 func (m PerpetualFuturesPosition) ProfitAndLossInMetrics(baseMetricsRate, quoteMetricsRate MetricsRateType) sdk.Int {
```

Comment:
I think InMetrics means in "usd" as the comment says.

If you modify it in "uusd" (micro metrics), change the function name.